New Data Source `azurerm_network_manager_network_group` (#22277)
hashicorp/terraform-provider-azurerm: website/docs/d/network_manager_network_group.html.markdown

---
subcategory: "Network"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_network_manager_network_group"
description: |-
  Get information about a Network Manager Network Group.
---

# azurerm_network_manager_network_group

Use this data source to access information about a Network Manager Network Group.

## Example Usage

```hcl
resource "azurerm_resource_group" "example" {
  name     = "example-resources"
  location = "West Europe"
}

data "azurerm_subscription" "current" {
}

resource "azurerm_network_manager" "example" {
  name                = "example-network-manager"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  scope {
    subscription_ids = [data.azurerm_subscription.current.id]
  }
  scope_accesses = ["Connectivity", "SecurityAdmin"]
  description    = "example network manager"
}

> 

Reviewer: Aren't we missing a `azurerm_network_manager_network_group` resource in this example?

data "azurerm_network_manager_network_group" "example" {
  name               = "example-group"
  network_manager_id = azurerm_network_manager.example.id
}
```

## Arguments Reference

The following arguments are supported:

> * `name` - (Required) Specifies the name which should be used for this Network Manager Network Group.

Reviewer: ```suggestion
* `name` - (Required) Specifies the name of the Network Manager Network Group.
```
